cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/Transform.java

```diff
@@ -17,7 +17,9 @@
 package co.cask.cdap.templates.etl.api;
 
 import co.cask.cdap.api.ProgramLifecycle;
+import com.google.common.reflect.TypeToken;
```

Comment:
Ya, since etl-api depends on cdap-api, that already brings in guava. But we need to not depend on it. Not sure if there is an easy way out. I will create a JIRA for this.